lifinance/jumper.exchange
# feat: personalised feature cards for users

Testing requirements:
- create a new user inside Strapi with your wallet address in Jumper User collection
- associate a specific card to your user in the Feature Card collection
- connect your wallet and you should view the new cards
